package.json

```diff
@@ -45,14 +51,18 @@
     "check-error": "^2.0.0",
     "deep-eql": "^5.0.1",
     "loupe": "^3.0.0",
-    "pathval": "^2.0.0"
+    "pathval": "^2.0.0",
+    "@types/check-error": "^1.0.3",
+    "@types/deep-eql": "^4.0.2",
+    "@types/pathval": "^1.1.2"
```

Comment:
if you're curious: these are prod dependencies since we export some of them in our public interface. so typescript consumers need to pull these types down too